components/Map.js

```diff
@@ -1430,15 +1439,15 @@ export function Map({ system,
           continue;
         }
 
-        const coords = stationIdsToCoordinates(stations, interchanges[interchangeId].stationIds);
+        const coords = stationIdsToMultiLineCoordinates(stations, interchanges[interchangeId].stationIds);
         if (coords.length > 1) {
```

Comment:
this will cause a bug since stationIdsToMultiLineCoordinates returns an array of arrays where the outer array can have length of one and be valid